tests/unit/converters/schema/halogen_docstring/cat.py

```diff
@@ -0,0 +1,14 @@
+"""The purpose of this file is to define a schema by importing halogen as an alias."""
+import halogen as hal
+
+
+class Cat(hal.Schema):
+    """Cat schema."""
+
+    @hal.Attr(hal.types.String())
+    def name(self):
+        """The name of the cat."""
+        return "Ginger"  # note: docstring above is currently not possible to parse.
```

Comment:
Is it hard to add? Can you parse `__doc__` on runtime?

Reply:
I updated the parser, now it can parse this docstring as well.